source/components/atoms/Input/Input.js

```diff
@@ -23,9 +24,12 @@ const inputText = styled(TextInput)`
 `;
 
 const Input = props => {
-  const { color } = props;
+  const { color, validation } = props;
+  const isValid = validation?.isRequired ? validateInput(props.value, validation?.rules)[0] : true;
```

Comment:
This check needs to be implemented in the other input components. SummaryList etc